ql/src/java/org/apache/hadoop/hive/ql/ddl/ShowUtils.java

```diff
@@ -145,45 +152,50 @@ public static String[] extractColumnValues(FieldSchema column, boolean isColumnS
           BinaryColumnStatsData binaryStats = statsData.getBinaryStats();
           values.addAll(Lists.newArrayList("", "", "" + binaryStats.getNumNulls(), "",
               "" + binaryStats.getAvgColLen(), "" + binaryStats.getMaxColLen(), "", "",
-              convertToString(binaryStats.getBitVectors())));
+              convertToString(binaryStats.getBitVectors()), ""));
```

Comment:
we do not need to put extra `""` here, it's safe to show NULL for a binary column?